src/deepsparse/pipeline.py

```diff
@@ -192,6 +193,12 @@ def __init__(
 
         self._batch_size = self._batch_size or 1
 
+        self.log(
```

Comment:
Hm now that I see this here, not sure if this is necessary since cores is static relative to a config change